### PR TITLE
[vm] Provide a benchmark instruction for macOS

### PR DESCRIPTION
**Pull request by runtian-zhou**
## Motivation

Provide a benchmark instruction for macOS.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

NA

### CODE REVIEW

**Comment by mengxu-fb** [comment]
Thanks for the README!

Maybe this is more related to dev-infra but I am just curious whether we will want to use more Rust-specific profilers going forward, like 
- using [inferno](https://github.com/jonhoo/inferno) to profile per-function execution time (with nice symbol resolving and even intra-function timing with proper annotations), and/or
- using [criterion](https://github.com/bheisler/criterion.rs) for checking performance regressions across commits.

File: language/benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- `Arith` and `call` benchmark in `diem/language/benchmark`
+
+The first item is a comprehensive benchmark of diem adapter, executor and storage that generates a block of p2p transactions and tries to execute and commit it to the DiemDB in local storage. The second item is a benchmark of Diem adapter only with a fake executor and an in-memory storage that executes randomly generated p2p transactions. The last item, although it’s still invoking Diem adapter, is mostly testing on the MoveVM’s ability of handling simple arithmetic operations and call stacks.
+
+## Step 3: Select the running process in Instrument.
+Open instrument and create a time profiler project.
+
+![](https://i.imgur.com/dbLht9f.png)
+
+Launch the benchmark target in the terminal, and select it in Instrument.
```
Comment (sblackshear) [inline language/benchmarks/README.md:25]: Can you add some more details on how to run each benchmark in a terminal? (are they scripts? do they use `cargo run`?)

**Review by sblackshear — APPROVED** [review]
Thanks!

**Pull request closed without merging**